queue/src/fiskaltrust.Middleware.Localization.QueueDE/JournalProcessorDE.cs

```diff
@@ -103,7 +103,7 @@ await foreach (var value in ProcessTarExportFromDatabaseAsync(request).Configure
             {
                 var result = new
                 {
-                    QueueDEList = _configurationRepository.GetQueueDEListAsync().Result
+                    QueueDEList = await _configurationRepository.GetQueueDEListAsync()
```

Comment:
Do we now why this has been done that way? This looks awful

Reply:
This is something that I'll test but I hope there was no reason 😅